docs/static/configuration.asciidoc

```diff
@@ -510,18 +510,45 @@ doesn't exist versus a field that's simply false. The expression `if [foo]` retu
 
 For more complex examples, see <<using-conditionals, Using Conditionals>>.
 
-[float]
+[discrete]
+[[date-math]]
+===== Date math in conditionals
+
+Date math in conditionals is not currently supported, but a workaround is available. 
+Put the date calculation in a field so that you can use the field reference in a conditional. 
+
+*Example* 
+
+This statement won't work: 
+
+`if "%{+HH}:%{+mm}" < "15:30" {`
```

Comment:
Maybe add the 

`add_field => {"string_compare" => "%{+HH}:%{+mm} is before 15:30"}`

part to this the broken example to give an equivalence between the working and non-working versions?